Containerfile

```diff
@@ -1,18 +1,18 @@
-FROM jupyter/r-notebook:r-4.2.2
+FROM jupyter/r-notebook:r-4.3.1
```

Comment:
Sweet.  Let's update this line as well to make sure we actually ship 4.3.1 - https://github.com/UCSB-PSTAT/base-rstudio/blob/main/Jenkinsfile#L22